scripts/build.sh

```diff
@@ -167,3 +172,12 @@ if [ -f docker-compose.override.yml ]; then
   echo "WARNING: This may add unwanted settings to the broker that could affect how your daemon runs."
   echo ""
 fi
+
+if [ "$LOCAL" == "true" ]; then
+  echo "Downloading Local Relayer Cert..."
+  # the path of this output is directly related to the SECURE_PATH that is set
+  # in the .env file
+  curl --silent -S --output ~/.sparkswap/secure/relayer-root.pem http://localhost:8080/cert
```

Comment:
we should consider renaming this relayer pem to something more obviously local/regtest.